README.md

```diff
@@ -14,6 +14,11 @@ This repository is a collaborative effort, so please feel free to make a pull re
 - [Getting Started](guides/Getting-Started.md): Download PureScript and build your first project
 - [Try PureScript](http://try.purescript.org): Try PureScript in your browser
 
+### Community
+
+- [Discourse](https://discourse.purescript.org/): Discuss PureScript in a forum setting
+- [Discord](https://discord.gg/sMqwYUbvz6): Chat about PureScript and attend video / audio meetups
```

Comment:
Alternately:

```suggestion
- [Discourse](https://discourse.purescript.org/): The PureScript Discourse instance
- [Discord](https://discord.gg/sMqwYUbvz6): The PureScript Discord server
```

Reply:
How about merging them both, so it reads "The PureScript Discourse Instance: discuss PureScript in a forum setting". Those familiar with the tool/platform will understand it right away. Those not will see what kinds of communication happen on it.